### PR TITLE
ES: I need a dumb topic name

### PR DESCRIPTION
**Pull request by maxii1996**

### CODE REVIEW

**Comment by maxii1996** [comment]
IDK why gives me error D:

**Comment by alama** [comment]
Because you changed some of the CSVs in the Files folder? you need to move them out

**Comment by maxii1996** [comment]
> Because you changed some of the CSVs in the Files folder? you need to move them out

I dont know what i've done D:

**Comment by alama** [comment]
No, you do not need to delete them, just rename/move them

**Comment by maxii1996** [comment]
but i dont know what the real name is it. And why it change it?
So, I'll start again

File: Files.TODO/co_058010.csv [hunk 1]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
 Title#0,"""『領域調査：異世界の残滓の解放』について"""
+=======
+Title#0,"""Inspec. Restos del Otro mundo"""
```
Comment (alama) [inline Files.TODO/co_058010.csv]: Why are there 2 `Title#0`?

**Comment by maxii1996** [comment]
Thanks you @alama :D

I'm really noob using GitHub i guess xD

**Review by alama — CHANGES_REQUESTED** [review]
Can not be merged until the CSV files in Files.TODO folder had been moved into suitable folders

**Comment by maxii1996** [comment]
@TheBlazter Por favor, dame una mano. Tengo miedo de hacer mas lio D:

**Comment by maxii1996** [comment]
ES: Creo que ya está? 
EN: I think is done?

**Review by alama — APPROVED** [review]

**Comment by TheBlazter** [comment]
Thanks @alama for the merge

**Pull request merged**